src/com/hotmail/kalebmarc/textfighter/main/Ui.java

```diff
@@ -112,6 +112,16 @@ public static void println() {
         print("\n");
     }
 
+    private static int hrLength = 66;
+    public static void printhr() {
+        println("-".repeat(hrLength));
+    }
+
+    // This function will reliably centre solely strings of an even length. I don't know of a way to fix this without varying hrLength. Do you? Couldn't varying hrLength solve the issue reliably as well?

```

Comment:
Here is the second one.

Reply:
I don't think there is a way to solve this. A variable hr length could work in theory, but would probably be more of a hassle than its worth, as the `printhr` function would need to know the string length as well, which would be weird. it isn't really noticeable anyways so I wouldn't worry about it